src/Compilers/CSharp/Test/Emit2/Semantics/RefStructInterfacesTests.cs

```diff
@@ -5316,5 +5316,937 @@ void verify(ModuleSymbol m)
                 Assert.False(u.AllowByRefLike);
             }
         }
+
+        [Fact]
+        public void ImplementAnInterface_01()
+        {
+            var src = @"
+interface I1
+{}
+
+ref struct S1 : I1
+{}
+";
+            var comp = CreateCompilation(src);
+
+            CompileAndVerify(comp, sourceSymbolValidator: verify, symbolValidator: verify).VerifyDiagnostics();
+
+            void verify(ModuleSymbol m)
+            {
+                var s1 = m.GlobalNamespace.GetMember<NamedTypeSymbol>("S1");
+                Assert.Equal("I1", s1.InterfacesNoUseSiteDiagnostics().Single().ToTestDisplayString());
+            }
+
+            CreateCompilation(src, targetFramework: TargetFramework.Net80, parseOptions: TestOptions.RegularNext).VerifyDiagnostics();
```

Comment:
Should we also test `RegularPreview` to ensure this keeps working in C#14+?

Reply:
> Should we also test RegularPreview to ensure this keeps working in C# 14+?

It is tested, that is the default option for our tests. Therefore, the ```CompileAndVerify``` covers that

Reply:
I've overlooked that, thanks.